lib/serializer.js

```diff
@@ -100,7 +100,7 @@ module.exports = class Serializer {
 
   asString (str) {
     if (typeof str !== 'string') {
-      if (str === null) {
+      if (str === null || str === undefined) {
```

Comment:
nit:

```suggestion
      if (str == null) {
```

Reply:
is it faster?

Reply:
It has its own bytecode instruction, so before JIT, it should be

Reply:
Oh wow seems to be a good bit quicker, this SO post outlines some numbers. Thanks for the input, I've learned something new!

https://stackoverflow.com/questions/7283577/speed-of-comparing-to-null-vs-undefined-in-javascript

```
➜  ~ node
Welcome to Node.js v20.5.1.
Type ".help" for more information.
> undefined == null
true
> undefined === null
false
```